source/frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -3,59 +3,62 @@ import './PropertyListView.scss';
 import styled from 'styled-components';
 
 import { StyledIconButton } from '@/components/common/buttons';
 import TooltipWrapper from '@/components/common/TooltipWrapper';
 import { Table } from '@/components/Table';
 import { TableSort } from '@/components/Table/TableSort';
 import * as API from '@/constants/API';
+import { MultiSelectOption } from '@/features/acquisition/list/interfaces';
 import { useApiProperties } from '@/hooks/pims-api/useApiProperties';
 import { useProperties } from '@/hooks/repositories/useProperties';
 import useLookupCodeHelpers from '@/hooks/useLookupCodeHelpers';
 import { useRouterFilter } from '@/hooks/useRouterFilter';
 import useDeepCompareEffect from '@/hooks/util/useDeepCompareEffect';
-import { IProperty } from '@/interfaces';
+import { Api_Property } from '@/models/api/Property';
 import { generateMultiSortCriteria } from '@/utils';
 import { toFilteredApiPaginateParams } from '@/utils/CommonFunctions';
 
 import { PropertyFilter } from '../filter';
-import { IPropertyFilter } from '../filter/IPropertyFilter';
+import { defaultPropertyFilter, IPropertyFilter } from '../filter/IPropertyFilter';
 import { SearchToggleOption } from '../filter/PropertySearchToggle';
 import { columns as columnDefinitions } from './columns';
 
-const defaultFilterValues: IPropertyFilter = {
-  searchBy: 'pinOrPid',
-  pinOrPid: '',
-  address: '',
-  page: '1',
-  quantity: '10',
-  latitude: undefined,
-  longitude: undefined,
-};
+const ownershipFilterOptions: MultiSelectOption[] = [
+  { id: 'isCoreInventory', text: 'Core Inventory' },
+  { id: 'isPropertyOfInterest', text: 'Property Of Interest' },
+  { id: 'isOtherInterest', text: 'Other Interest' },
+  { id: 'isDisposed', text: 'Disposed' },
+];
+
+//interface SortParams {}
 
 const PropertyListView: React.FC<React.PropsWithChildren<unknown>> = () => {
   const { getByType } = useLookupCodeHelpers();
-  const tableFormRef = useRef<FormikProps<{ properties: IProperty[] }> | undefined>();
+  const tableFormRef = useRef<FormikProps<{ properties: Api_Property[] }> | undefined>();
 
   const municipalities = useMemo(() => getByType(API.ADMINISTRATIVE_AREA_TYPES), [getByType]);
 
   const columns = useMemo(() => columnDefinitions({ municipalities }), [municipalities]);
 
   // We'll start our table without any data
-  const [data, setData] = useState<IProperty[] | undefined>();
+  const [data, setData] = useState<Api_Property[] | undefined>();
 
   // Filtering and pagination state
-  const [filter, setFilter] = useState<IPropertyFilter>(defaultFilterValues);
+  const [filter, setFilter] = useState<IPropertyFilter>(defaultPropertyFilter);
 
   const [pageSize, setPageSize] = useState(10);
   const [pageIndex, setPageIndex] = useState(0);
   const [pageCount, setPageCount] = useState(0);
   const [totalItems, setTotalItems] = useState(0);
-  const [sort, setSort] = useState<TableSort<IProperty>>({});
+  const [sort, setSort] = useState<TableSort<any>>({});
```

Comment:
are they <any> 's in this file intentional?

Reply:
yeah I was matching the other definitions. I made it Api_Property instead